### DISCUSSION TITLE
代码助手

### DISCUSSION THREAD

**Discussion by cemcke** [InternLM/Tutorial#300] [project]
项目名：代码助手
项目背景：欢迎大家参与提升Intern-chat-7b模型的代码生成能力的项目。通过这个项目，我们的目标是使Intern-chat-7b模型在代码生成问题上表现比基座更好，并且让团队成员熟悉大模型在垂直方向落地的全流程。
项目工作内容：
1.	数据集的选择与预处理：负责选择合适的数据集，并进行必要的预处理，以提供给模型进行微调与知识库外挂。
2.	提示词设计：探索和设计有效的提示词策略，以便模型在生成代码时能够更准确地理解用户需求并给出正确的生成结果。
3.	代码知识库外挂：将代码知识库与模型结合，使模型能够从中获取更多的代码知识并应用于生成过程。
4.	代码数据微调：对代码数据进行微调，以使模型在代码生成任务上表现更出色。
5.	模型量化：尝试多种量化策略，以降低模型的资源消耗同时保持性能。
6.	前端页面设计：设计友好易用的前端页面，让用户可以方便地使用提示词与模型进行交互，并展示生成的代码结果。
7.	平台部署：将模型成功部署在openxlab平台上，以便用户能够方便地使用和评估模型效果。
项目预期目标：
1.	模型在未见过的代码问题上比基座效果更好，能够生成更准确、高质量的代码。
2.	团队成员能够熟悉大模型在垂直方向落地的全流程，包括数据处理、模型微调、知识库外挂、前端设计和平台部署等方面的工作。
预期团队成员4人
还差3人

**Comment by leochiang** [comment]
7班有人组队嘛？

**Comment by SiliconSisyphus** [comment]
你好，请问满员了吗

**Comment by cemcke** [comment]
已经满员了哈
